common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -90,6 +90,13 @@ export enum SignalType {
 	ClientLeave = "leave",
 }
 
+export enum SignalSource {
+	/**
+	 * Signal sent by server on behald of external caller (through brodcast-signal service endpoint).
```

Comment:
```suggestion
	 * Signal sent by server on behalf of external caller (through brodcast-signal service endpoint).
```

Reply:
i would like to see a design document before we alter the protocol